master/internal/db/postgres_trial.go

```diff
@@ -161,11 +161,11 @@ WHERE id = $1`, id, restartCount); err != nil {
 
 // fullTrialSummaryMetricsRecompute recomputes all summary metrics for a given trial.
 func (db *PgDB) fullTrialSummaryMetricsRecompute(
-	ctx context.Context, tx *sqlx.Tx, trialID int,
-) error {
-	_, err := tx.ExecContext(ctx, `
+	ctx context.Context, tx *sqlx.Tx, trialID int, jsonPath, table string,
```

Comment:
it should work the same if the table is always "metrics" and we instead set the parition_type column but feel free to leave this out